modin/backends/pandas/query_compiler.py

```diff
@@ -2722,7 +2722,7 @@ def groupby_agg_builder(df, by=None, drop=False, partition_idx=None):
                 )
                 internal_by_cols = internal_by_df.columns
 
-                external_by = by.columns.difference(internal_by)
+                external_by = by.columns.difference(internal_by).unique()
```

Comment:
Here we need a set of columns without duplications, however new version of pandas returns *all* of the elements that are not presented at the `difference` argument, including duplicated ones
```python
>>> pandas125.Index(["a", "a", "a"]).difference([])
Index(['a'], dtype='object')
>>> pandas130.Index(["a", "a", "a"]).difference([])
Index(['a', 'a', 'a'], dtype='object')
```
so calling explicit `unique()` here to get a non-duplicating set